src/Http/Controllers/Traits/BreadRelationshipParser.php

```diff
@@ -115,4 +118,16 @@ protected function relationToLink(Model $item, DataType $dataType)
 
         return $item;
     }
+
+    protected function getField($item, $relationMethod)
+    {
+        $relationBuilder = $item->$relationMethod();
+
+        // original field named was used so we'll return that
+        if ($relationBuilder instanceof BelongsTo) return $relationBuilder->getForeignKey();
+
+        // While adding the relationship we used getRelationName() as field name,
+        // so we'll return that now too.
+        if ($relationBuilder instanceof BelongsToMany) return $relationBuilder->getRelationName();
```

Comment:
I completely understand ignoring some of the other Laravel/Eloquent relationship types (morphTo, etc), but why only BelongsTo and BelongsToMany? What about their inverses (hasOne, hasMany)? what about the others (i.e. hasManyThrough)?